diagnostic.md

```diff
@@ -48,11 +50,15 @@ Record your responses inside the fenced code blocks below each question.
     value `'123'` inside a Route?
 
     ```md
-    <!-- your response here -->
+    [params.movie_id - 1]
     ```
 
 1.  Inside a template, how do we reference data provided by a Route?
 
     ```md
-    <!-- your response here -->
+    HTMLbars:
+
+    {{#each thing as |thinger|}}
+      {{thinger.key}}
+    {{/each}}
```

Comment:
The key here is what you're referencing as `thing`. That actually needs to be `model`. The model is what the Route provides us to hook into that data.